### PR TITLE
Permission SSP-63

### PR DESCRIPTION
**Pull request by ArtemSokoliuk**

### CODE REVIEW

File: SSP/ssp_project/templates/index.html [hunk 1]
```diff
@@ -20,7 +20,15 @@
 
     <p>
         ITA SoftServe
-    </p>
+    </p
```
Comment (Fedorof) [inline SSP/ssp_project/templates/index.html]: ???
Just do this:

    <p>ITA SoftServe

You don't have to use closing tag for `<p>`

**Comment by Fedorof** [comment]
Please, rebase to resolve conflicts

**Comment by ArtemSokoliuk** [comment]
Will be done, I almost wrote behave tests...want to try with phantomjs in vagrant

**Comment by ArtemSokoliuk** [comment]
created another pull request

**Comment by Fedorof** [comment]
So this one should be declined?

**Comment by ArtemSokoliuk** [comment]
Yes, it should be declined

**Pull request closed without merging**